framework/src/problems/FEProblemBase.C

```diff
@@ -1600,6 +1604,8 @@ FEProblemBase::reinitDirac(const Elem * elem, THREAD_ID tid)
         _scalar_zero[tid].resize(FORTYTHIRD, 0);
         _zero[tid].resize(max_qpts, 0);
         _grad_zero[tid].resize(max_qpts, RealGradient(0.));
+        _grad_phi_zero[tid].resize(max_qpts, {0., 0., 0.});
+        _grad_test_zero[tid].resize(max_qpts, {0., 0., 0.});
```

Comment:
@lindsayad this init here is ugly but I can't figure out how to init with RealVectorValue...
suggestions?